weblab/entities/forms.py

```diff
@@ -6,7 +6,12 @@
 from accounts.models import User
 from core import visibility
 
-from .models import EntityFile, ModelEntity, ProtocolEntity
+from .models import (
+    Entity,
```

Comment:
You can get rid of this import now. There are a few other flake8 errors in the build too; I'll not go through them all.